influx-writer/influx-writer.py

```diff
@@ -117,16 +118,19 @@ def write(self, listen_dicts):
 
     def start(self):
         self.log.info("InfluxWriterSubscriber started")
+        print("InfluxWriterSubscriber started")
 
         self.register(SUBSCRIBER_NAME)
         while True:
             try:
                 count = self.subscriber()
             except NoSubscriberNameSetException as e:
                 self.log.error("InfluxWriterSubscriber has no subscriber name set.")
+                print("InfluxWriterSubscriber has no subscriber name set. Exiting.")
                 return
             except WriteFailException as e:
                 self.log.error("InfluxWriterSubscriber failed to write: %s" % str(e))
+                print("InfluxWriterSubscriber failed to write, skipping this batch of data.")
```

Comment:
Done! Only errors get logged via the function to stderr. I haven't made a function for the start message because it seemed like overkill to me. Should I make one?